router/package_request.go

```diff
@@ -59,7 +62,7 @@ func newPackageRequest(args newPackageRequestArgs) (*packageRequest, error) {
 
 	// Only go out to github to get the matched SHA if the matched SHA is
 	// necessary.
-	if isGoGetRequest(args.req) {
+	if isGoGetRequest(args.req) || isGitRequest(parts) {
```

Comment:
Yeah. Would have been a mess to change it in both places. This is a cleaner merge bc its exactly one changed line.